service/controller/v3/resource/vnetpeeringcleaner/create.go

```diff
@@ -0,0 +1,43 @@
+package vnetpeeringcleaner
+
+import (
+	"context"
+
+	"github.com/giantswarm/microerror"
+
+	"github.com/giantswarm/azure-operator/service/controller/v3/key"
+)
+
+// EnsureCreated ensure that vnetpeering resource are deleted,
+// since they are no longer in use in this version.
+func (r Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
+	r.logger.LogCtx(ctx, "level", "debug", "message", "ensuring delete host vnetpeering")
```

Comment:
I would say it would be better and more correct in one of these two ways. 

- `ensuring deletion of host vnetpeering`
- `deleting host vnetpeering`